<launch>
  <node ns="apm" pkg="mavros" type="driver.py" name="mavros" output="screen" args="--device=udp:192.168.1.5:57812">
  </node>
  <node ns="mosaic" pkg="mavros" type="rosnode.py" name="mosaic_node" output="screen" args="--prefix=/apm/">
  </node>
</launch>


